src/Controller/CocktailController.php

```diff
@@ -72,7 +72,7 @@ public function submitName(Request $request): Response
         ]);
     }
 
-    private function cocktailNameRequest(CocktailName $cocktailName): string
+    private function cocktailNameRequest(CocktailName $cocktailName)
```

Comment:
You will refactor this logic into separate service so it doesn't really matter but later don't return string or array type as it would attract more bugs later. It's better to use Data Transfer Object (DTO, google it) to transfer response ant error data as response and then only in higher level (controller for example) parse error message and result data.
For example, CocktailNameData would have array property, which holds returned data and error property with error message. Either one of them will always be not set so it will be easier to check for result.